be/src/exprs/bitmap_functions.cpp

```diff
@@ -162,21 +162,30 @@ StatusOr<ColumnPtr> BitmapFunctions::bitmap_or(FunctionContext* context, const s
 StatusOr<ColumnPtr> BitmapFunctions::bitmap_and(FunctionContext* context, const starrocks::Columns& columns) {
     RETURN_IF_COLUMNS_ONLY_NULL(columns);
 
-    ColumnViewer<TYPE_OBJECT> lhs(columns[0]);
-    ColumnViewer<TYPE_OBJECT> rhs(columns[1]);
-
+    std::vector<ColumnViewer<TYPE_OBJECT>> list;
+    list.reserve(columns.size());
+    for (const ColumnPtr& col : columns) {
+        list.emplace_back(col);
+    }
     size_t size = columns[0]->size();
     ColumnBuilder<TYPE_OBJECT> builder(size);
     for (int row = 0; row < size; ++row) {
```

Comment:
suggest to add two optimization:
1. if any bitmap is empty, then return empty bitmap early;
2. chose the bitmap whose cardinality is the bigest as the initial accumulating result, then sort the rest bitmaps in the ascending order of their cardinalties,  the merge the sorted bitmaps one by one into  initial accumulating result; after each merging, return quickly as the result is already empty.

Reply:
1. No need to do 1, bitmap_and will do this
```
BitmapValue& BitmapValue::operator&=(const BitmapValue& rhs) {
    switch (rhs._type) {
    case EMPTY:
        reset();
        break;
```

And empty Bitmap is not a common scenario.

2.Maybe RoaringBitmap already do this

```
void roaring_bitmap_and_inplace(roaring_bitmap_t *x1,
                                const roaring_bitmap_t *x2) {
    if (x1 == x2) return;
    int pos1 = 0, pos2 = 0, intersection_size = 0;
    const int length1 = ra_get_size(&x1->high_low_container);
    const int length2 = ra_get_size(&x2->high_low_container);

```

Reply:
Hi @satanson , after I add the sort logic to this patch as the latest code show, I make a benchmark:
```
select bitmap_and(ss_ticket_number,ss_addr_sk,ss_cdemo_sk,ss_customer_sk,ss_item_sk,ss_sold_time_sk,ss_hdemo_sk,ss_sold_date_sk,ss_quantity,ss_promo_sk,ss_store_sk)
from store_sales_bitmap;
````
,  I found that it will make the function slower. Do you think it's still necessary to add this sort logic?